source/dofs/dof_renumbering.cc

```diff
@@ -1280,21 +1293,19 @@ namespace DoFRenumbering
       }
     else
       {
-        //this is not a distributed Triangulation. Traverse coarse cells in the
-        //normal order
-        for (cell = dof_handler.begin (0); cell != dof_handler.end (0); ++cell)
+        // this is not a distributed Triangulation. Traverse coarse cells in the
+        // normal order
+        for (typename DoFHandler<dim>::level_cell_iterator cell = dof_handler.begin (0);
```

Comment:
huh, I know we used a ``level_cell_iterator`` before, but that doesn't make any sense to me, it should be a ``cell_iterator`` or not?

Reply:
Interesting point. Let me leave this for a separate PR, given that it changes functionality.

Reply:
See #5689.